### ISSUE TITLE
Where did SQLite.Net.Platform.XamarinAndroid and SQLite.Net.Platform.XamarinIOS go?

### ISSUE THREAD

**Issue by nSolvePR** [oysteinkrog/SQLite.Net-PCL#234]
Updated to 3.1.0 from 3.0.5 and can't build now as SQLite.Net.Platform.XamarinAndroid has disappeared from my package list. Suggestions please

**Comment by tbalcom** [comment]
I'm having the same problem. I've rolled back to 3.0.5 for now.

**Comment by oysteinkrog** [comment]
Hmm, these nuget packages were removed some time ago.
Are you saying that an install of 3.1 does not properly add these packages (when adding to an Android project)?

**Comment by nSolvePR** [comment]
iOS and Android both have the same issue.  Note all I did was UPGRADE the packages and these packages disappeared.  I tried removing the SQLite packages and adding them back but the Platform specific packages did not reappear

**Comment by oysteinkrog** [comment]
Hm, I think it might have been a problem with me not updating all version entries in the nuspecs.
I've uploaded 3.1.1 now, please test.

**Comment by tbalcom** [comment]
I've updated from 3.1.0 to 3.1.1 but the problem still remains.

**Comment by nSolvePR** [comment]
Ta. I updated to 3.1.1 but it did not add the packages.

**Comment by oysteinkrog** [comment]
@mattleibow Do you know why this could be happening, it's possible that it is caused by the nuget split.

**Comment by Sylix** [comment]
Same here with SQLitePlatformWinRT, nuget: SQLite.Net.Async-PCL. Ok 3.0.5, not working both 3.1.0 and 3.1.1

Problems:
**using SQLite.Net.Platform.WinRT;**

var connectionFactory = new Func<SQLiteConnectionWithLock>(() => new SQLiteConnectionWithLock(new **SQLitePlatformWinRT()**, new SQLiteConnectionString(databasePath, storeDateTimeAsTicks: false)));

**Comment by nSolvePR** [comment]
Any news on a solution to this issue? Thanks

**Comment by oysteinkrog** [comment]
Hmm, I'm having trouble reproducing this error.
I created class libraries (xamaring ios and android). Installing 3.0.5 and the above code (connectionfactory) works fine.
Then I upgraded to 3.1.1 and it still compiles.
Do you guys know of a way to cleanly reproduce?

**Comment by tbalcom** [comment]
I've had success removing the previous versions via the NuGet GUI, deleting any folder with 'sqlite' in the packages folder, then adding version 3.1.1 again directly using the NuGet GUI.

**Comment by nSolvePR** [comment]
My 'fix' was to add the source files directly to the project. Will try tbalcom suggestion

**Comment by nSolvePR** [comment]
@tbalcom  What is the 'NuGet GUI' you refer to?

**Comment by tbalcom** [comment]
With NuGet we either get the packages using the console (CLI) or Visual Studio (GUI).  The GUI is the one you get when you right-click the Solution in Visual Studio and select 'Manage NuGet Packages for Solution'.  The CLI is the one you get when you select (Tools > NuGet Package Manager > Package Manager Console).

**Comment by nSolvePR** [comment]
Thanks, as I only ever use Xamarin Studio I'll try the Nuget console to see if that does something different to using the XS package manager. As removing all, deleting any sql folders in my packages directory did not fix the issue.

**Comment by alexrainman** [comment]
SQLite.Net.Platform not there anymore.

**Comment by mgjhl** [comment]
@oysteinkrog I have tried uninstalling all SQLite packages and re-installing them again - Did not resolve the issue.

**Comment by rofiqsetiawan** [comment]
@mgjhl  Did you tried to uninstalling all SQLite packages and reinstalling with this order?

1. SQLite.Net.Async-PCL
2. SQLite.Net.Platform.XamarinAndroid

it works, at least in my case

**Comment by Sylix** [comment]
I have to do this to work, SQLite.Net.Platform.WinRT.

Working:
SQLite.Net.Async PCL 3.0.5
Dependencies: SQLite.Net-PCL

Not working:
SQLite.Net.Async PCL 3.1.1
Dependencies: SQLite.Net.Core-PCL


If i install SQLite.Net PCL 3.1.1 (it was a SQLite.Net.Async PCL 3.0.5 dependency), i have SQLite.Net.Platform working again.

I believe SQLite.Net.Platform shoudl be moved from SQLite.Net PCL to SQLite.Net.Core-PCL.

**Comment by jesusgarza** [comment]
I had the same issue with SQLite.Net.Platform.XamarinIOS. Uninstalled and reinstalled SQLite.net PCL (which installed SQLite.Net-PCL and SQLite.Net.COre-PCL dlls) and then SQLite.Net.Async PCL (which installed SQLite.Net.Async-PCLdll). That solved it.

**Comment by drewlederman** [comment]
Any updates on this? I've tried every suggestion and still no SQLite.Net.Platform... Thanks!

**Comment by AlexanderOsokin** [comment]
Manually add following lines to .csproj file and it will work.

    <Reference Include="SQLite.Net.Platform.XamarinIOS.Unified">
      <HintPath>..\packages\SQLite.Net-PCL.3.1.1\lib\Xamarin.iOS10\SQLite.Net.Platform.XamarinIOS.Unified.dll</HintPath>
    </Reference>

**Comment by econner20** [comment]
Any updates on this? I am trying to use SQLite.Net PCL (3.1.1) in a PCL library created in Visual Studio 2015. NuGet installs the SQLite.NET Core and SQLite.Net PCL but the SQLite.Net.Platform namespace is missing.

**Comment by Manne990** [comment]
It is gone from Nuget... Is there a solution or work around?
https://www.nuget.org/packages/SQLite.Net.Platform.XamarinIOS

**Comment by oysteinkrog** [comment]
Yes, install the main package, it will add the needed references.

**Comment by jagl16** [comment]
It still says there's a missing assembly reference when trying to use SQLitePlatform Android and iOS.

This are the packages i'm using. 
SQLite.Net.Async-PCL 3.1.1
SQLite.Net.Core-PCL 3.1.1

**Comment by Sylix** [comment]
jagl16 Install SQLite.Net-PCL too. All the 3 packages.

**Comment by fritzlim** [comment]
I too had the same issue - SQLite.Net.Platform.XamarinIOS was missing.

I solved it by installing the 3.1.1 package, and then using the code in the iOS Project section from this recipe https://developer.xamarin.com/guides/xamarin-forms/working-with/databases/.

That recipe doesn't use SQLite.Net.Platform.XamarinIOS.

**Comment by chauncat** [comment]
Any update?

I am not able to pull down 3.1.1 even after I uninstall in reinstall.  I do not get the platform specific packages.  Nuget has message on the Xamarin Android and IOS page stating that 
> The owner has unlisted this package. This could mean that the package is deprecated or shouldn't be used anymore.

[https://www.nuget.org/packages/SQLite.Net.Platform.XamarinIOS](url)
[https://www.nuget.org/packages/SQLite.Net.Platform.XamarinAndroid/](url)

**Comment by alexrainman** [comment]
I just started to use REALM and i am happy with the results.

**Comment by chauncat** [comment]
REALM is still a little to new for us right now.

**Comment by BoubakrEchieb** [comment]
I think there is no need for SQLite.Net.Platform.XamarinAndroid and SQLite.Net.Platform.XamarinIOS, 
just use the sqlite-net-pcl Nuget package... as in detailed in this link : 
https://developer.xamarin.com/guides/xamarin-forms/working-with/databases/

**Comment by leankitryan** [comment]
Nothing I did with Xamarin Studio 6.0.2 or the 6.1 beta would let me bring in the `SQLite.Net.Platform.XamarinAndroid` dependency. I had to open my solution in Visual Studio and use its NuGet package manager to get bring in the correct platform dependencies.

**Comment by viktorvillalobos** [comment]
The same, here, any solution?

**Comment by dpybus** [comment]
Same here. Any Solution?

**Comment by RodBarnes** [comment]
@oysteinkrog You last stated "Yes, install the main package, it will add the needed references."  But having installed SQLite.Net and SQLite.Net.Async, the .Xamarin.iOS and .Xamarin.Android packages remain missing.  What is the go-forward for these packages?

**Comment by stopiccot** [comment]
And in a year 2017 problem is still not fixed :(

**Comment by AnubhutiChopra** [comment]
@oysteinkrog  We appreciate your work. Why you have stopped updating this repository. Please provide SQLite.Net.Platform.XamarinAndroid and SQLite.Net.Platform.XamarinIOS and update this repo as people wanted to use this library for PCL applications. Thanks :)

**Comment by jatovar** [comment]
still not solved?

**Comment by tbalcom** [comment]
Just use https://github.com/praeclarum/sqlite-net. This project seems to be mostly abandoned.
